packages/examples/src/demo-servients/bridge-cli.ts

```diff
@@ -128,7 +127,7 @@ If the file 'bridge-servient.conf.json' exists, that configuration is applied.
 Options:
   -v, --version           display node-wot version
   -f, --configfile=file   load configuration from specified file
-  -p, --password=secret   pass the password for the Oracle trust store
+  -p, --password=secret   pass the password for the trust store
```

Comment:
It feels that this option was pretty related to the Oracle implementation, should we really keep it?

Reply:
@wiresio ?